hr_expense_cancel/models/hr_expense.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class HrExpenseSheet(models.Model):
+    _inherit = "hr.expense.sheet"
+
+    def action_cancel(self):
+        for sheet in self:
+            account_move = sheet.account_move_ids
```

Comment:
Since we have a `X2many` field, variable name should be plural
```suggestion
            account_moves = sheet.account_move_ids
```